vql/parsers/sdjournal/watcher_linux.go

```diff
@@ -112,6 +112,12 @@ func (self *JournalWatcherService) StartMonitoring() {
 		return
 	}
 
+	_,err = self.journal.Previous()
+	if err != nil {
+		scope.Log("Failed to seek tail of journal: %v", err)
```

Comment:
Maybe change the message a bit to distinguish it from the previous log message.

Reply:
Any suggestions? Both SeekTail() and Previous() are called to achieve the same thing (seeking tail) so I wasn't sure of a good way to differentiate here.

Reply:
Not sure, maybe something like `Failed to set read pointer to previous journal entry`